driver/core/kv_cmd.c

```diff
@@ -519,11 +519,11 @@ int _kv_nvme_exist(kv_nvme_t* nvme, const kv_pair* kv, int qid) {
                 LEAVE();
                 return ret;
         }
-        pthread_spin_unlock(&qpair->sq_lock);
 
         while(!io_sequence.is_completed) {
                 spdk_nvme_qpair_process_completions(qpair, 0);
         }
```

Comment:
the loop can exist all the time? ask considering lock release.

Reply:
This is a synchronous interface, when this request is processed, it will exit the loop. This is consistent with other interfaces of the original code, such as _kv_nvme_store，_kv_nvme_retrieve，_kv_nvme_delete